back/engines/commercial/project_management/spec/services/user_role_service_spec.rb

```diff
@@ -1,93 +1,99 @@
-require "rails_helper"
+require 'rails_helper'
 
 describe UserRoleService do
-  let(:service) { UserRoleService.new }
+  let(:service) { described_class.new }
 
-  describe "can_moderate_project?" do
-    let(:project) { create(:project) }
+  describe 'can_moderate_project?' do
+    let(:project) { create :project }
 
-    it "permits project moderators" do
+    it 'permits project moderators' do
       expect(service.can_moderate_project? project, create(:project_moderator, projects: [project])).to be_truthy
     end
 
-    it "denies other project moderators" do
+    it 'denies other project moderators' do
       expect(service.can_moderate_project? project, create(:project_moderator, projects: [create(:project)])).to be_falsey
```

Comment:
I consider `be_falsey` to weak. The method returns a boolean, so I would write:
```suggestion
      expect(service.can_moderate_project? project, create(:project_moderator, projects: [create(:project)])).to be false
```